ansible/roles/ensure_crc/tasks/main.yml

```diff
@@ -62,3 +62,13 @@
         regexp: 'timeout 10 crc console --credentials'
         line: >
           eval $(timeout 10 crc console --credentials | awk -F"'" '/kubeadmin/{print $2}')
+    - name: unset GOFLAGS for vscode ssh plugin to work
+      ansible.builtin.lineinfile:
+        path: "~/.bashrc"
+        regexp: "GOFLAFS"
+        line: "unset GOFLAFS"
+    - name: use existing cluster for go tests
```

Comment:
we removed supprot for this
we nolonger support using existing clusters only envtest for the nova-operator at least.

in general I'm not sure we should modify the bashrc with etiehr of these

the ssh plugin shoudl work with out the goflags changes

did you get any specific error without this?